content/6_Advanced/SCC.mdx

```diff
@@ -34,7 +34,7 @@ In the first phase, the algorithm performs a depth-first search (DFS) on the ori
 
 In the second phase, the algorithm conducts another DFS, this time on the transposed graph, where all edges are reversed. The DFS processes nodes according to the order defined by the stack from the first phase. By reversing the edges and following this specific order, each DFS run in this phase can identify all vertices of one SCC before moving on to the next. This ensures that the SCCs are accurately and efficiently determined.
 
-We know that the transposed graph will have the same SCCs as the original because reversing an edge doesn't change the reachability between two nodes. Since each SCC is maximal and disjoint, reversing edges doesn't break any connections in the original graph.
+We know that the transposed graph will have the same SCCs as the original because reversing an edge doesn't change the reachability between two componets. Since each SCC is maximal and disjoint, reversing edges doesn't break any connections in the original graph.
```

Comment:
uh
i don't think this means what you think it means
perhaps you meant "within" instead of "between"?
also typo lol

Reply:
my bad

Reply:
updated